tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -163,4 +163,77 @@ public function userCreatesAPublicLinkShareWithSettings(string $user, TableNode
 			)
 		);
 	}
+
+	/**
+	 * @When /^user "([^"]*)" has created the following link share:$/
+	 *
+	 * @param string $user
+	 * @param TableNode|null $body
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userHasCreatedALinkShareWithSettings(string $user, TableNode  $body):void {
+		$this->userCreatesAPublicLinkShareWithSettings($user, $body);
```

Comment:
we had a hard time refactoring this :sweat_smile:  . so let's not use step functions into another step defs 
```suggestion
		<helper-function>
		<save-response>
```